proto/debugpb.proto

```diff
@@ -148,3 +151,10 @@ message ListFailPointsResponse {
 
     repeated Entry entries = 1;
 }
+
+message GetMetricsRequest {
+}
```

Comment:
should add a content-type to return text or protobuf format data.

Reply:
But the response is a string.

Reply:
Could we support text first？